__test__/index.test.js

```diff
@@ -0,0 +1,31 @@
+
+import request from 'supertest';
+import app from '../server';
+
+
+let server;
+let agent;
+beforeEach((done) => {
```

Comment:
Can we make this a beforeAll?

Reply:
okay, this is to reduce the no of starting and stopping the server on each test case